app/models/bot/alegre.rb

```diff
@@ -398,7 +399,11 @@ def self.transcribe_audio(pm)
 
   def self.media_file_url(pm)
     # FIXME Ugly hack to get a usable URL in docker-compose development environment.
-    url = (ENV['RAILS_ENV'] != 'development' ? pm.media.file.file.public_url : "#{CheckConfig.get('storage_endpoint')}/#{CheckConfig.get('storage_bucket')}/#{pm.media.file.file.path}")
+    if pm.class == TemporaryProjectMedia
```

Comment:
Suggestion / non-blocker: You could rewrite as `pm.is_a?(TemporaryProjectMedia)`.